lib/gcpspanner/daily_chromium_histogram_metrics_test.go

```diff
@@ -168,6 +168,21 @@ func insertSampleDailyChromiumHistogramMetrics(
 	}
 }
 
+func insertGivenSampleDailyChromiumHistogramMetrics(
```

Comment:
Same thing with removing the existing method and using yours
```suggestion
func insertTestDailyChromiumHistogramMetrics(
```

Reply:
Updated function names and existing references. 🙂